packages/atomic/src/pages/examples/insights.html

```diff
@@ -0,0 +1,62 @@
+<!DOCTYPE html>
+<html dir="ltr" lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0, minimum-scale=1.0, maximum-scale=5.0" />
+    <title>Coveo Atomic: Insights Panel example</title>
+
+    <script type="module" src="/build/atomic.esm.js"></script>
+    <script nomodule src="/build/atomic.js"></script>
+    <link rel="stylesheet" href="/themes/coveo.css" />
+    <script type="module">
+      async function main() {
+        await customElements.whenDefined('atomic-svc-insight-interface');
+        const insightInterface = document.querySelector('atomic-svc-insight-interface');
+        await insightInterface.initialize({
+          accessToken: 'xxbb74d0cd-9374-45b9-ae5c-f106e2cc0795',
```

Comment:
Is this okay to put public ? For `searchuisamples` it's okay since we know that it's all public content, and it's well understood by everyone that adds content in there.

If not, the key needs to be deleted ASAP. Because by now all manner of bots across the internet have stored it ;) 

Also, we'll need something more solid long term, that's available in a production org, as opposed to dev.

It can stay like that for now (as long as the key can be made public).

Reply:
I'm not sure, I'll ask Luc later but delete for now!